docker/scripts/entrypoint.sh

```diff
@@ -47,18 +47,19 @@ do
         # Systemd and OpenRC (and possibly others) both create such a
         # cgroup. To avoid the aforementioned bug, we symlink "foo" to
         # "name=foo". This shouldn't have any adverse effect.
-        echo $SUBSYS | grep -q ^name= && {
-                NAME=$(echo $SUBSYS | sed s/^name=//)
-                ln -s $SUBSYS $CGROUP/$NAME
+        echo "$SUBSYS" | grep -q ^name= && {
+                # shellcheck disable=SC2001
+				NAME=$(echo "$SUBSYS" | sed s/^name=//)
```

Comment:
there's a strange mix of tabs/spaces going on here

Reply:
yeah, this file uses both :( Fixed this one in particular and resolved the conflict